### DISCUSSION TITLE
Shouldn't it be a dev dependency?

### DISCUSSION THREAD

**Discussion by RemyMachado** [iamvishnusankar/next-sitemap#631] [Q&A] [answered]
The `readme.md` indicates `yarn add next-sitemap`, but shouldn't it be installed as a dev dependency? If not, why?

**Comment by iamvishnusankar** [comment, marked as answer]
@RemyMachado It's totally fine to install `next-sitemap` as dev dependency if you're only using it to generate static sitemaps. 

However, a lot of people were using it for server sitemap generation. The initial documentations suggested to install it as dev dep, however users were having internal server issues after deployment because a lot of hosted services were only installing dependencies (production mode).

So, I changed the documentation to make it as a main dependency. But, you're totally free to move it to dev dep if your use case is only static sitemap generation.
